MapboxNavigation/NavigationViewController.swift

```diff
@@ -628,11 +565,11 @@ extension NavigationViewController: RouteMapViewControllerDelegate {
         self.delegate?.navigationViewController?(self, didSelect: route)
     }
     
-    @objc public func navigationMapView(_ mapView: NavigationMapView, shapeFor routes: [Route]) -> MLNShape? {
+    public func navigationMapView(_ mapView: NavigationMapView, shapeFor routes: [Route]) -> MLNShape? {
```

Comment:
I've annotated the whole class with `@objcMemebers`, so we don't need to mark every function with `@objc`